external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -296,7 +296,8 @@ class KafkaTestUtils(withBrokerProps: Map[String, Object] = Map.empty) extends L
     props.put("replica.socket.timeout.ms", "1500")
     props.put("delete.topic.enable", "true")
     props.put("offsets.topic.num.partitions", "1")
-    props.putAll(withBrokerProps.asJava)
+    // props.putAll(withBrokerProps.asJava)
+    withBrokerProps.foreach{ case (k, v) => props.put(k, v) }
```

Comment:
Is this related?

Reply:
https://github.com/scala/bug/issues/10418. I see.

Reply:
yes, the change to the putAll is similar as the solution provided in scala/bug#10418